### ISSUE TITLE
touch not working

### ISSUE THREAD

**Issue by supertopoz** [transcriptic/react-map-interaction#4]
This is a really fantastic!! However, when objects in the zoom view are clicked with a mouse they act as expected, but with a touch screen there is no event listener for touch as. Is there an intention to add touch support? 

Update 1: I narrowed down the issue. Touch is supported but once the first touch support has fired it gets locked to that type. For example if the first touch is movement then all subsequent touches are for for movement. If the first touch fires a click like event all subsequent touches are clicks and movement is lost. 

Update 2: To overcome the above issues. In the source code I changed the onTouchDown function. Then in my React component I added both a click handler and a touch handler. 

Source for react-map-interaction

```
  onTouchDown(e) {
    e.preventDefault();
    //  e.stopPropagation();
    this.setPointerState(e.touches);
  }
```

React 

```
onTouchStart={() => this.toggle(clickedHeader, true)}
onClick={() => this.toggle(clickedHeader, true)}
```

**Comment by scottyantipa** [comment]
Thanks for the issue @supertopoz . We'll take a look. Also feel free to put up a PR if you feel up to it.

**Comment by scottyantipa** [comment]
Resolved in this commit: https://github.com/transcriptic/react-map-interaction/commit/dc2376b55590ddf54509cce0ff98ff3cae043dae

Will be published with version 0.0.4

**Comment by scottyantipa** [comment]
@supertopoz Version 0.0.4 is released with this bug fixed. Thanks for the issue. I would recommend using `onTouchEnd` typically, instead of `onTouchStart`. If the user clicks and drags on your target element, then `onTouchStart` will be called at the beginning of the drag gesture. However, MapInteraction will stop the propagation of the `onTouchEnd` event if it was triggered by the user dragging. Both can come in handy, but it is likely more common to only trigger the action when the use is not dragging (e.g. a button click).

Please post what you're working on so we can see some use cases for MapInteraction. We'd love to see a screen shot.
